Solutions/OracleDatabaseAudit/Data Connectors/Connector_OracleDatabaseAudit.json

```diff
@@ -52,7 +52,7 @@
     "instructionSteps": [
         {
             "title": "",
-            "description": ">This data connector depends on a parser based on a Kusto Function to work as expected. [Follow these steps](https://aka.ms/sentinel-OracleDatabaseAudit-parser) to create the Kusto Functions alias, **OracleDatabaseAuditEvent**",
+            "description": "**NOTE:** This data connector depends on a parser based on a Kusto Function to work as expected which is deployed as part of the solution. To view the function code in Log Analytics, open Log Analytics/Microsoft Sentinel Logs blade, click Functions and search for the alias Oracle Database Audit and load the function code or click [here](https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/OracleDatabaseAudit/Parsers/OracleDatabaseAuditEvent.txt), on the second line of the query, enter the hostname(s) of your Oracle Database Audit device(s) and any other unique identifiers for the logstream. The function usually takes 10-15 minutes to activate after solution installation/update.",
```

Comment:
Hi @v-prasadboke please update the note here and remove the part for hostname and other identifiers, keep the note inline with the parser instructions, thanks!